rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```
@@ -1113,6 +1113,57 @@ SecRule REQUEST_HEADERS:Referer "@rx ^[^\.]*?(?:['\*\?\x5c`][^\n/]+/|/[^/]+?['\*
             setvar:'tx.rce_score=+%{tx.critical_anomaly_score}',\
             setvar:'tx.inbound_anomaly_score_pl2=+%{tx.critical_anomaly_score}'"
 
+
+#
+# -=[ Rule 932207 ]=-
+#
+# Sibling of 932205 targeting fragments in the Referer header.
+#
+# The last chain prevents FPs against the "Scroll to text fragment" browser feature
+# (https://wicg.github.io/scroll-to-text-fragment/).
+#
+# Regular expression generated from regex-assembly/932207.ra.
+# To update the regular expression run the following shell script
+# (consult https://coreruleset.org/docs/development/regex_assembly/ for details):
+#   crs-toolchain regex update 932207
+#
+SecRule REQUEST_HEADERS:Referer "@rx ^[^#]+\#[^#]+" \
```

According to the RFC, `#` is only allowed as the delimiter for the fragment. So just look for the first `#` and treat everything afterwards as fragment.
```suggestion
SecRule REQUEST_HEADERS:Referer "@rx #.*" \
```

---

Since we modified this, we need to modify the `TX:0` value, otherwise the payloads won't match. I changed the `TX:0` value, the tests are still failing and need to be fixed. This was the previous regex: https://regex101.com/r/YMChGH/1